pages/KeyConceptsInOppia.html

```diff
@@ -102,7 +102,7 @@ <h2>
     screenshot shows what two different playthroughs of the same exploration
     look like, when placed side-by-side:
   </p>
-  <img src="static/images/threeBallsSideBySide.png" width="100%">
+  <img src="static/images/threeBallsSideBySide.png" width="100%" alt="A blue text-box displaying an explaination followed by three balls arranged in the red-blue-yellow order in a straight horizontal line. Below which is an 'add item' button allowing the learner to enter all the combinations he's come up with. Adjacent to it on the right is a blue box asking the learner to select from two radio buttons, title 'yes' and 'no' respectively, based on if the present answer was the same as their first guess.">
```

Comment:
explanation is misspelled.

Below which is --> This is followed by
he's --> they've
title --> labelled
if the --> whether the